src/functionaltests/java/com/ericsson/ei/status/StatusSteps.java

```diff
@@ -0,0 +1,89 @@
+/*
+   Copyright 2019 Ericsson AB.
+   For a full list of individual contributors, please see the commit history.
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package com.ericsson.ei.status;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Ignore;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.TestPropertySource;
+
+import com.ericsson.ei.handlers.MongoDBHandler;
+import com.ericsson.ei.handlers.StatusHandler;
+import com.ericsson.ei.utils.FunctionalTestBase;
+import com.fasterxml.jackson.databind.JsonNode;
+
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+
+@Ignore
+@TestPropertySource(properties = {
+        "spring.data.mongodb.database: StatusSteps",
+        "missedNotificationDataBaseName: StatusSteps-missedNotifications",
+        "rabbitmq.exchange.name: StatusSteps-exchange",
+        "rabbitmq.consumerName: StatusSteps-Consumer" })
+public class StatusSteps extends FunctionalTestBase {
```

Comment:
Should the steps classes maybe have something related to test in the name? I don't remember if we have said anything about this but it seems to be the case in every other functional test :)

Reply:
Majority of the step files seem to not have "Test" in the name.

Reply:
Fair enough, I think it was perhaps the runner classes that needed to have test in the name to be recognized by Cucumber

Reply:
Renamed